quipucords/scanner/network/connect.py

```diff
@@ -142,6 +142,7 @@ def run_with_result_store(self, result_store):
         serializer = SourceSerializer(self.scan_task.source)
         source = serializer.data
 
+        optional_products_status = self.scan_job.create_product_status_dict()
```

Comment:
This is unnecessary as the connection flow doesn't run the playbook, it just checks for authentication.